### ISSUE TITLE
Support for Winston 3 transports options (eg custom formats, defaultMeta)

### ISSUE THREAD

**Issue by lululeon** [timberio/timber-js#49]
Hi, looking at the code [here](https://github.com/timberio/timber-js/blob/6ebcfc9c1d10df24abd9525aca1c2784e22646fc/packages/winston/src/winston.ts#L9), there does not appear to be support for passing down options to the underlying winston 3 transports (eg custom formats). A change to the constructor to accept transport options and pass these down would be great, eg:

```
  public constructor(private _timber: Timber, transportOpts) {
    super(transportOpts);
  }
```

Relatedly on [this line](https://github.com/timberio/timber-js/blob/6ebcfc9c1d10df24abd9525aca1c2784e22646fc/packages/winston/src/winston.ts#L40) we only forward `info.message`, so we would be discarding the "defaultMeta" metadata fields that winston-based loggers might be using to provide logging context. Would it be OK to just forward the entire `info` object, and then let users select only the `.message` portion for forwarding (they can do so with a custom format, which the first point addresses).

**Comment by keeross** [comment]
Agree, would be better to send full `info` object, sending only `message` part strips down all the meta, that makes timber via winston simply useless.

**Comment by leebenson** [comment]
`defaultMeta` is available in 0.34.1 (it was previously merged --- thanks @hawx -- but unreleased.), and I've bumped the devDependency of Winston to ensure tests don't fail due to the key not being recognised in Typescript (which affected Winston < 3.2.1)
